### ISSUE TITLE
Pandoc citation output is (???) 

### ISSUE THREAD

**Issue by calimammoth** [retorquere/zotero-better-bibtex#1028]
Hi,

Most of my pandoc citations come out correctly. However, many are showing up as ??? when i view my markdown pandoc work in marked2

Any idea?

**Comment by retorquere** [comment]
What is your citekey pattern set to, what version of BBT do you have, and what does the key look like in Zotero?

**Comment by retorquere** [comment]
closing for inactivity

**Comment by calimammoth** [comment]
What is your citekey pattern set to, what version of BBT do you have, and what does the key look like in Zotero? 

cite key format?? [auth:lower][year]
BBT version- seems to update every 12 hours or so. ha.
the key in zotero?- hmmm- 
<img width="218" alt="screen shot 2018-07-16 at 14 09 07" src="https://user-images.githubusercontent.com/40992014/42760274-d87dda3c-8901-11e8-8b3d-17e10f02b94a.png">

**Comment by retorquere** [comment]
Even if BBT updates frequently (and it has the past few weeks), I still want to know the version that you are currently on.

How are you generating the `???` keys? CAYW, export, drag-drop?

**Comment by calimammoth** [comment]
Ah okay. It's 5.0.170. 

I use the CAYW zotpicker applet. Most citations come out just fine.

**Comment by retorquere** [comment]
Can you right-click the reference that gives you this output and select BBT->send error report?

**Comment by retorquere** [comment]
(it works best if you first pick this references using CAYW and then right-click it and send the report)

**Comment by retorquere** [comment]
Also, try clicking http://localhost:23119/better-bibtex/cayw?format=pandoc . What do you see in your browser after you've picked that reference?

**Comment by calimammoth** [comment]
Hi @retorquere thanks for trying to help me.

I didn't get anything back from the browser one. And nothing happens when I right click a reference in the zotpicker.

Is there just something wrong with the cite keys potentially? Is it possible to automatically recode them all?

**Comment by retorquere** [comment]
I can tell you more when I have the debug report. Make sure zotero is running, click the link, pick a reference, and then send a BBT error report. Without I have to resort to guessing what's wrong.

**Comment by calimammoth** [comment]
Hey man, sorry, I think i've figured it out. I think that it's just innaccurate cite keys. Not sure why some where being cited in a different pattern to what I had set in BBT. Some were showing journal article etc aswell. When I've changed them to the standard pattern they now show up properly.

Sorry about that. I'm very new to all of this stuff. And I don't know anyone who uses anything other than Microsoft Word for everything. I'm the rogue one.
